imports/plugins/core/hydra-oauth/register.js

```diff
@@ -21,6 +21,17 @@ Reaction.registerPackage({
     description: "Oauth Login Provider Page",
     workflow: "hydraOauthLogin",
     template: "hydraOauthLoginForm"
+  }, {
+    route: "/not-found",
+    name: "not-found",
+    label: "not-found",
+    meta: {
+      noAdminControls: true,
+      oauthLoginFlow: true
+    },
+    description: "Not Found Page",
+    workflow: "hydraOauthLogin",
+    template: "notFound"
```

Comment:
IDP's Not Found route definition. Here it's using the existing/default template. It can be updated to use a custom template as needed